GitUI/FormSignatureInfo.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using GitCommands.Utils;
+
+namespace GitUI
+{
+    public partial class FormSignatureInfo : GitExtensionsForm
+    {
+        public FormSignatureInfo()
+        {
+            InitializeComponent();
+            Translate();
+        }
+
+        public void ShowGpgMessage(string message)
+        {
+            gpgTextBox.Text = EnvUtils.ReplaceLinuxNewLinesDependingOnPlatform(message);
```

Comment:
GPG tag window must not select all text when opened and the textbox must be readonly:
![image](https://user-images.githubusercontent.com/4403806/30855407-43ea0912-a2f8-11e7-9cda-46bee92801db.png)
![image](https://user-images.githubusercontent.com/4403806/30855415-4d54053e-a2f8-11e7-909c-268c25222047.png)

Reply:
Ok for readonly, but text can be selectable if user want to copy it.
You've asked to copy this text in the past, before it was a simple dialog.

Reply:
It must remain selectable so users can copy text from it. 
It should not open with all text selected

Reply:
yes, it already has this behavior fixed on last commit.